optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -127,7 +127,9 @@
         <maven.jdbc.driver.jar/>
         <maven.jdbc.username/>
         <maven.jdbc.password/>
-        <maven.jdbc.url>jdbc:h2:</maven.jdbc.url>
+        <!-- Relative path in H2 JDBC URL must be explicit.
+             See http://www.h2database.com/javadoc/org/h2/api/ErrorCode.html#c90011. -->
+        <maven.jdbc.url>jdbc:h2:tcp://localhost/./target/persistence-test</maven.jdbc.url>
```

Comment:
This is only for QA testing with an actual db, right?

Reply:
I did not introduce that change, afaik, btw.